src/sentry/incidents/serializers/alert_rule.py

```diff
@@ -243,6 +254,17 @@ def _validate_query(self, data):
         ):
             dataset = data["dataset"] = Dataset.Metrics
 
+        if features.has(
+            "organizations:ddm-experimental",
+            self.context["organization"],
+            actor=self.context.get("user", None),
+        ):
+            column = get_column_from_aggregate(data["aggregate"])
```

Comment:
Had to re-run this code here, since I couldn't access the requested dataset from the `validate_aggregate` function.